src/Organisms/EnhancedList/EnhancedList.jsx

```diff
@@ -0,0 +1,145 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import { css } from '@styled-system/css'
+import Paper from '@material-ui/core/Paper'
+import { Download } from 'styled-icons/boxicons-regular/Download'
+import { Columns } from 'styled-icons/boxicons-regular/Columns'
+import { Search } from 'styled-icons/material/Search'
+import { Plus } from 'styled-icons/boxicons-regular/Plus'
+
+import { useTranslation } from '../../Atoms/Trans'
+import Pagination from '../../Molecules/Pagination'
+import BaseExport from '../../Molecules/Export'
+import BaseEditColumns from './EditColumns'
+import Menu from '../../Atoms/Menu'
+import Tooltip from '../../Atoms/Tooltip'
+import ButtonGroup from '../../Atoms/ButtonGroup'
+import IconButton from '../../Atoms/IconButton'
+import Button from '../../Atoms/Button'
+import Input from '../../Atoms/Input'
+import InputAdornment from '../../Atoms/Input/InputAdornment'
+
+import TableList from '../../Molecules/TableList'
+
+const Toolbar = styled.div`
+  display: flex;
+  justify-content: space-between;
+`
+
+const SearchWrapper = styled.div`
+  display: flex;
+  flex-grow: 1;
+
+  & > * {
+    ${css({ mx: 's' })};
+  }
+`
+
+const SearchInput = styled(Input)`
+  flex-grow: 1;
+`
+
+const ActionGroup = styled(ButtonGroup)`
+  ${(css({ mx: 's' }))}
+`
+
+const EnhancedList = ({
+  Export = BaseExport,
+  ExportProps = {},
+  EditColumns = BaseEditColumns,

```

Comment:
If we can override the whole component we don't need to override the props.

Reply:
`EnhancedList` needs to know about the anchor element that opens/closes `Export` and the prop `columns` is automatically passed down to `EditColumns`. I don't think it would be easier to use that way.